src/pages/Login.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from 'react';
+import users from '../data/user.json'; 
+import loginsvg from '../assets/images/login-svg.svg'; 
+import { toast } from 'react-toastify';
+import { useNavigate } from 'react-router';
+
+function Login() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const navigate = useNavigate();
+
+  const handleLogin = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const user = users.find((user) => user.username === username);
+    if (user) {
+      if (user.password === password && user.username === username) {
```

Comment:
as that check already has happened while extracting the user 
```suggestion
      if (user.password === password) {
```